.github/actions/matrix-fan-in/action.yml

```diff
@@ -0,0 +1,48 @@
+name: "Matrix Fan-in"
+description:
+  "Collect all JSON files from the matrix and combine them into a single file
+  (matrix-fan-in.json)."
+
+inputs:
+  path:
+    description: Must match matrix-fan-out `path`.
+    default: ./matrix-fan-out
+  fan-out-files:
+    description: |-
+      Optionally filter which fan-out files to include in the fan-in.
+      syntax: find(1) -name
+    default: matrix*.json
+  pattern:
+    description: |-
+      Optionally filter which matrix results to include e.g. "color=blue"
+      syntax: @actions/glob
+    default: "*"
+  shell:
+    default: bash -euxo pipefail {0}
```

Comment:
Missing description which is required according to GitHub's schema (https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputsinput_iddescription)

Reply:
The selftest passes, so it doesn't seem to be very required, to me.
I'll fix, though.

No need to duplicate comment -- I'll always scan for other instances of the same issue.